src/sidebar/components/Annotation/AnnotationLicense.js

```diff
@@ -5,15 +5,15 @@ import { Icon, Link } from '@hypothesis/frontend-shared';
  */
 export default function AnnotationLicense() {
   return (
-    <div className="hyp-u-padding--top hyp-u-border--top u-font--small u-line-height">
+    <div className="hyp-u-padding--top hyp-u-border--top text-sm leading-none">
```

Comment:
The `hyp-u-` classes here could easily be swapped to relevant TW classes, but I wanted to keep this PR focused.